titles/admin-rhdh/title-admin.adoc

```diff
@@ -18,6 +18,9 @@ include::installation/assembly-install-rhdh-ocp.adoc[leveloffset=+1]
 
 include::modules/auth/assembly-enabling-authentication.adoc[leveloffset=+1]
 
+// Running RHDH behind a proxy
+include::getting-started/proc-running-rhdh-behind-a-proxy.adoc[leveloffset=+1]
```

Comment:
```suggestion
include::admin/proc-running-rhdh-behind-a-proxy.adoc[leveloffset=+1]
```

Is this the right path? I couldn't see this new section from the preview website:

![image](https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/assets/593208/8bf17d15-9c59-42a5-9e92-566aac48a022)

Reply:
Updated, Thanks